go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/gobuffalo/meta
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/gobuffalo/envy v1.6.7
-	github.com/gobuffalo/flect v0.0.0-20181108195648-8fe1b44cfe32
+	github.com/gobuffalo/flect v0.0.0-20181104133451-1f6e9779237a
```

Comment:
This looks reverted. 

Can you rm go.* and rebuild them with go mod tidy?

Reply:
Sure.